src/Rootfs.jl

```diff
@@ -420,6 +430,7 @@ function supported_platforms()
         Windows(:i686),
         Windows(:x86_64),
     ]
```

Comment:
There was a part of me that had a knee-jerk reaction to this list of platforms and almost commented "Just use `supported_platforms()` instead of listing these out like this".  lol.

Reply:
haha! Hopefully with this PR those pains are reduced!